cardano-crypto-tests/src/Test/Crypto/Vector/Vectors.hs

```diff
@@ -0,0 +1,140 @@
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
+
+{-# HLINT ignore "Use bimap" #-}
+
+module Test.Crypto.Vector.Vectors
+  ( defaultSKey,
+    defaultMessage,
+    defaultEcdsaSignature,
+    defaultSchnorrSignature,
+    signAndVerifyTestVectors,
+    wrongEcdsaVerKeyTestVector,
+    wrongSchnorrVerKeyTestVector,
+    wrongLengthMessageHashTestVectors,
+    ecdsaVerKeyAndSigVerifyTestVectors,
+    schnorrVerKeyAndSigVerifyTestVectors,
+    ecdsaMismatchMessageAndSignature,
+    schnorrMismatchMessageAndSignature,
+    verKeyNotOnCurveTestVectorRaw,
+    wrongLengthVerKeyTestVectorsRaw,
+    ecdsaWrongLengthSigTestVectorsRaw,
+    schnorrWrongLengthSigTestVectorsRaw,
+    ecdsaNegSigTestVectors
+  )
+where
+
+import Cardano.Binary (FromCBOR)
+import Cardano.Crypto.DSIGN
+  ( DSIGNAlgorithm (SigDSIGN, SignKeyDSIGN, VerKeyDSIGN),
+    EcdsaSecp256k1DSIGN,
+    SchnorrSecp256k1DSIGN,
+  )
+import Data.ByteString (ByteString)
+import Test.Crypto.Vector.SerializationUtils
+  ( HexStringInCBOR (..),
+    sKeyParser,
+    sigParser,
+    encodeUtf8,
+    vKeyParser,
+  )
+
+defaultSKey :: forall d. (FromCBOR (SignKeyDSIGN d)) => SignKeyDSIGN d
+defaultSKey = sKeyParser "B7E151628AED2A6ABF7158809CF4F3C762E7160F38B4DA56A784D9045190CFEF"
+
+defaultMessage :: ByteString
+defaultMessage = encodeUtf8 "243F6A8885A308D313198A2E03707344A4093822299F31D0082EFA98EC4E6C89"
+
+defaultEcdsaSignature :: SigDSIGN EcdsaSecp256k1DSIGN
+defaultEcdsaSignature = sigParser "c0730606584a92b4a979fdbfbb89a6b304827ab5084e55f61f6c1fbf36cf359b49a8e128aee4bba7fa5b8b0491ba2425aa97a2af668cb4c54fb68dfae8a675c8"
+
+defaultSchnorrSignature :: SigDSIGN SchnorrSecp256k1DSIGN
+defaultSchnorrSignature = sigParser "8c29c80b168a3b7a6fa90bb17785e25205ff09bc01616115b00f81a17d1eb6dbea1a0c02aa138a7b2af1557fb762d0d9e6d8742adff4013c7d064612ebc27f37"
+
+-- These vectors contains secret key which first signs the given message and verifies using generated signature and derived vKey
+signAndVerifyTestVectors :: forall d. (FromCBOR (SignKeyDSIGN d)) => [(SignKeyDSIGN d, ByteString)]
+signAndVerifyTestVectors =
+  map
+    (\(sk, m) -> (sKeyParser sk, encodeUtf8 m))
```

Comment:
`encodeUtf8` is redundant also consider switching to list comprehensions instead of a `map`, this will allow you to remove the hlint hint prevention too:
`{-# HLINT ignore "Use bimap" #-}`

Reply:
Yeah. Great Thanks.